### PR TITLE
Fix Wrong Variable Name In setup_symlinks.py

### PR DESCRIPTION
**Pull request by TheBrokenRail**
Fixes

```
Traceback (most recent call last):
  File "/usr/local/bin/appimage-builder", line 8, in <module>
    sys.exit(__main__())
  File "/usr/local/lib/python3.8/dist-packages/appimagebuilder/__main__.py", line 44, in __main__
    invoker.execute(commands)
  File "/usr/local/lib/python3.8/dist-packages/appimagebuilder/invoker.py", line 29, in execute
    command()
  File "/usr/local/lib/python3.8/dist-packages/appimagebuilder/commands/setup_symlinks.py", line 46, in __call__
    if preserve_file.samefile(bin):
  File "/usr/lib/python3.8/pathlib.py", line 1113, in samefile
    other_st = os.stat(other_path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not builtin_function_or_method
```

### CODE REVIEW

File: appimagebuilder/modules/setup/generator.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def _patch_interpreted_executables(self, executables, patcher: ExecutablesPatche
         for executable in interpreted_executables:
             allowed = True
```
Comment (azubieta) [inline appimagebuilder/modules/setup/generator.py:109]: I guess that it would be better to put tho whole check in a function so it's easier to read

**Review by azubieta — CHANGES_REQUESTED** [review]

File: appimagebuilder/modules/setup/generator.py (hunk 1 above)
Comment (TheBrokenRail) [reply appimagebuilder/modules/setup/generator.py]: Done!

File: appimagebuilder/utils/finder.py [hunk 2]
```diff
@@ -160,3 +160,27 @@ def _run_check(self, check_function, path):
             passed = check_function(path)
             self.cache[key] = passed
             return passed
+
+    @staticmethod
+    def list_does_not_contain_file(file_list: [pathlib.Path], file: pathlib.Path):
+        allowed = True
+        for item in file_list:
+            if str(item) == str(file):
```
Comment (azubieta) [inline appimagebuilder/utils/finder.py:168]: why a string conversion  is required if both variables should be of Path type?

File: appimagebuilder/utils/finder.py [hunk 3]
```diff
@@ -160,3 +160,27 @@ def _run_check(self, check_function, path):
...
+                allowed = False
```
Comment (azubieta) [inline appimagebuilder/utils/finder.py:169]: you could just return from here

File: appimagebuilder/utils/finder.py (hunk 2 above)
Comment (TheBrokenRail) [reply appimagebuilder/utils/finder.py]: Sorry, I haven't coded in Python in a while. Fixed!

File: appimagebuilder/utils/finder.py (hunk 3 above)
Comment (TheBrokenRail) [reply appimagebuilder/utils/finder.py]: Fixed!

**Comment by azubieta** [comment]
Many Thanks!

**Pull request merged**